projects/ks89/angular-modal-gallery/src/lib/components/previews/previews.component.ts

```diff
@@ -245,86 +229,81 @@ export class PreviewsComponent extends AccessibleComponent implements OnInit, On
   }
 
   /**
-   * Private method to init previews based on the currentImage and the full array of images.
-   * The current image in mandatory to show always the current preview (as highlighted).
-   * @param InternalLibImage currentImage to decide how to show previews, because I always want to see the current image as highlighted
-   * @param InternalLibImage[] images is the array of all images.
+   * Indicates if the previews 'left arrow' should be displayed or not.
+   * @returns 
    */
-  private initPreviews(currentImage: InternalLibImage, images: InternalLibImage[]): void {
```

Comment:
Same here: for simplification and edge-case handling, I propose that the whole logic be processed uniformly in a single method (setIndexesPreviews)